src/cmds/cloud/updateSchema.js

```diff
@@ -0,0 +1,73 @@
+/* eslint-disable no-console */
+import yargs from 'yargs';
+import { Client } from '@cesarbr/knot-cloud-sdk-js';
+import options from './options';
+
+const updateSchema = (args) => {
+  const client = new Client({
+    protocol: args.protocol,
+    hostname: args.server,
+    port: args.port,
+    pathName: args.pathName,
+    id: args['client-id'],
+    token: args['client-token'],
+  });
+
+  client.on('ready', () => {
+    client.updateSchema([
+      {
+        sensorId: args['sensor-id'],
+        valueType: args['value-type'],
+        unit: args.unit,
+        typeId: args['type-id'],
+        name: args.name,
+      },
+    ]);
+  });
+  client.on('updated', () => {
+    client.close();
+  });
+  client.on('error', (err) => {
+    console.log(err);
+    console.log('Connection refused');
+  });
+  client.connect();
+};
+
+yargs
+  .command({
+    command: 'update-schema [sensor-id] [value-type] [unit] [type-id] [name]',
+    desc: 'Update a thing schema',
+    builder: (_yargs) => {
+      _yargs
+        .options(options)
```

Comment:
You can create an option to the json file path

Reply:
Done.